app/src/main/java/ch/epfl/qedit/view/quiz/QuestionFragment.java

```diff
@@ -34,7 +34,14 @@ public View onCreateView(
 
         final QuizViewModel model =
                 new ViewModelProvider(requireActivity()).get(QuizViewModel.class);
-
+        System.out.println("=====================================");
+        System.out.println("=====================================");
+        System.out.println("=====================================");
+        System.out.println(model);
+        System.out.println("=====================================");
+        System.out.println("=====================================");
+        System.out.println("=====================================");
+        System.out.println("=====================================");
```

Comment:
I suppose it's for debug purpose, don't forget to delete that before merging with master.